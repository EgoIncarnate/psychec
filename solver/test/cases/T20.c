void f() {
  T1 a = 1.0;
  T2 b;
}
