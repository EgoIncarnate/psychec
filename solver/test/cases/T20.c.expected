typedef uintptr_t T2;
typedef double T1;
